syfertext/language.py

```diff
@@ -101,12 +101,13 @@ def make_doc(self, text: Union[str, String, StringPointer]):
         are stored in a Doc object which is then returned.
         """
 
+        location_id = text.location.id
+
```

Comment:
What if text is of type `str` or PySyft `String`, it won't have the `location` attribute.